helix-core/src/test/java/org/apache/helix/manager/zk/TestZKUtil.java

```diff
@@ -37,13 +37,15 @@
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
+
 public class TestZKUtil extends ZkUnitTestBase {
   private static Logger LOG = LoggerFactory.getLogger(TestZKUtil.class);
 
   String clusterName = CLUSTER_PREFIX + "_" + getShortClassName();
 
   @BeforeClass()
-  public void beforeClass() throws Exception {
+  public void beforeClass()
+      throws Exception {
```

Comment:
Nit, this line is unnecessary. Style format file is updated in master branch. You may want to re-import the file in your Intellij so this line won't be formatted.